x-pack/plugins/alerting/server/alerts_service/resource_installer_utils.ts

```diff
@@ -19,6 +19,7 @@ export const VALID_ALERT_INDEX_PREFIXES = [
   '.alerts-',
   '.internal.preview.alerts-',
   '.preview.alerts-',
+  '.reindexed-v8-',
```

Comment:
I wouldn't include this in here. I think we'll need more changes in the alerts service and resource installer stuff to handle the new prefix so we should create a separate PR (I pushed some changes to start here: https://github.com/elastic/kibana/pull/204819).

Reply:
Ok, moving it there